prog/weave-kube/weave-daemonset-k8s-1.6.yaml

```diff
@@ -92,6 +92,11 @@ spec:
               cpu: 10m
           securityContext:
             privileged: true
+          env:
+          - name: MY_NODE_NAME
```

Comment:
`HOSTNAME` is also set by `bash`.

For other use-cases there is a command-line option `--node-name`.